pkg/client/v1/nimbleos/ns_o_t_p_type.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Hewlett Packard Enterprise Development LP
+package nimbleos
+
+// Golang package for NsOTPType Enum.
+
+type NsOTPType string
```

Comment:
I believe the generator has an acronym list to improve symbol style conversions.  Adding "OTP" to the list would improve this file name, changing it from ns_o_t_p_type.go to ns_otp_type.go.